<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pulsePage</class>
 <widget class="QWidget" name="pulsePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../Resources.qrc">:/icons/icons/pulse_explanation.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_refreshBackground">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh Background</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_blueLaserSpot">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>211</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>At position of blue dot</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="open_shutter__blueDot">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Open Shutter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="close_shutter___blueDot">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Close Shutter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pulse_button___blueDot">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Pulse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>121</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>At current Position</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="open_shutter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Open Shutter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="close_shutter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Close Shutter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pulse_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Pulse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
   <zorder></zorder>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>280</y>
     <width>121</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>T</string>
        </property>
        <property name="buddy">
         <cstring>T_spinBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="T_spinBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>minDeltaT</string>
        </property>
        <property name="buddy">
         <cstring>minDeltaT_spinBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="minDeltaT_spinBox"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>93</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>show green dot </string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_thunderstorm">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>mark spots for pulses </string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>181</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>clear</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="open_shutter__blueDot_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Start thunderstorm</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
